controller/network/routesender.go

```diff
@@ -148,6 +126,61 @@ attendance:
 	return peerData, nil, nil
 }
 
```

Comment:
Above you've got counters for invalid terminator and misconfigured terminator, but I only see invalid being used here. I think you could mark invalid when you're deleting and misconfigured when you're setting to failed